src/UserForm.js

```diff
@@ -104,11 +110,11 @@ class UserForm extends React.Component {
       request,
     } = this.props;
 
-
     const id = user.id;
     const name = getFullName(user);
     const barcode = user.barcode;
-    const editMode = get(request, 'id');
+    const editMode = !!request;
```

Comment:
Maybe rename `editMode` to `isEditable`? "mode" implies something other than a boolean to me.

Reply:
@zburke, makes sense, done!